### PR TITLE
Issue 1083: Jobs tab

### LINKED ISSUE #1083: User Management: Jobs tab
### Description
We want to refresh this view a little bit and move it to "User management" view as tab.
https://sketch.cloud/s/mrZ88/all/3d-repo-monday/jobs

### Goals
- [x] Admins can add new jobs to teamspace
- [x] Admins can change current color
- [x] Admins can choose new colors
- [x] Admins can filter jobs by name
- [x] Admins can sort jobs by name or color

### Tasks
- [x] Jobs list component
- [x] FAB for adding new jobs
- [x] Color picker component

### PR DESCRIPTION
**Pull request by simplypixi**
This fixes #1083 

The same situation like before. I had to create this branch from #1085  branch. #1085  PR is not merged yet, but I need these changes to continue my work.

#### Description
- Added color picker
- Added new user management tab (Jobs)

### CODE REVIEW

**Comment by carmenfan** [comment]
@simplypixi  thanks 😄 

A few quick ones I spot:
- The + button is disabled if the user is out of license. The User should be able to add jobs regardless if they have maxed our their license
- Can we have a OK/Cancel putting on the colour picker? (or ok and a cross)
- Can we bold "Job" as well? 
image
- The colour picker and the job entries seems to move around depending on where I clicked on the colour picker
filter



We'll send you a detailed code review as soon as we can!

File: frontend/components/account/js/account-user-management.component.ts [hunk 1]
```diff
@@ -41,199 +41,225 @@ const TABS = {
...
+	/**
+	 * Get teamspace projects list
+	 * @param teamspaceName
+	 */
+	public getTeamspaceProjects(teamspaceName: string): object[] {
+		if (!teamspaceName) {
 			return [];
 		}
+		// TODO: Handle request
+		return [];
```
Comment (Charence) [inline frontend/components/account/js/account-user-management.component.ts]: Is this being used? It looks incomplete.

**Comment by simplypixi** [comment]
> The colour picker and the job entries seems to move around depending on where I clicked on the colour picker
It's happening because of active sorting by color. ;) Should I disable this for color update?

**Comment by carmenfan** [comment]
@simplypixi  I think we can get rid of the colour sort all together, as it doesn't really mean much to anyone :)

File: frontend/components/account/js/account-user-management.component.ts (hunk 1 above)
Comment (simplypixi) [reply frontend/components/account/js/account-user-management.component.ts]: Done ;)

**Comment by simplypixi** [comment]
@carmenfan Fixed ;)

**Comment by carmenfan** [comment]
@simplypixi 
Almost there! just a few tiny UI related remarks:
- Can we change the OK and Cancel styling to be like this on the colour picker?
image

- Can we add a footer so it's consistent with other tabs? (maybe say something like "Manage Jobs")
image

- Two things: 
    1. there should only 1 colour picker up at any time 
    2. the picker can be obscured if the job is on a lower row (not sure what's the best remedy but currently I can't click ok on it)
filter

- How hard is it to make it so the colour changes as you drag around the colour picker? (At the moment the colour only changes after the user clicks, which is quite counter intuitive considering other existing applications tends to do the drag thing)
filter

**Comment by simplypixi** [comment]
@carmenfan Fixed! ;)

**Comment by simplypixi** [comment]
@carmenfan Fixed and ready to merge ;)

**Pull request merged**